LDAR_Sim/src/ldar_sim.py

```diff
@@ -101,9 +102,15 @@ def run_simulation(self):
             self._output_manager._update_ts_row_w_emis_info(
                 new_row=new_row, ts_emis_info=ts_emis_info, ts_emis_rep_info=ts_emis_rep_info
             )
-            self._output_manager._update_ts_row_w_methods_info(
-                new_row=new_row, ts_methods_info=ts_methods_info
-            )
+            if first_day:
```

Comment:
Instead of doing if first_day and then calling this with an explicit true, can you just pass first day in as the include upfront cost value and then just have the if to change it to false